_i18n/en/general/about.md

```diff
@@ -5,7 +5,7 @@ There's so many cool things to do in the world, and so many things in life to wo
 
 In that context, we realise that every option we add means a choice not made. And a choice not made, is a decision that we push to the end-user. (Here's a great [blog post on why developers shouldn't do that](http://neugierig.org/software/blog/2018/07/options.html) lightly.) We promise to do our best to make those decisions that create a great experience. Because - did you decide yet what you're going to eat tonight?
 
-At the same time: don't worry. We'll be open to a new feature, option or preference - whenever there's a case to make for offering our users flexibility. We're working on tags for podcasts, and recently introduced local feeds, for example. So if you've got your own idea, feel free to [share your feature request on our forum](https://forum.antennapod.org/c/feature-request)!
+At the same time: don't worry. We'll be open to a new feature, option or preference - whenever there's a case to make for offering our users flexibility. We have recently introduced tags for podcasts and support for adding local folders, for example. So if you've got your own idea, feel free to [share your feature request on our forum](https://forum.antennapod.org/c/feature-request)!
```

Comment:
Hmm, this paragraph will get outdated very soon. Not sure if we should really mention things about the ongoing development here.